src/Bouncer.php

```diff
@@ -373,52 +369,44 @@ public function ownedVia($model, $attribute = null)
      * Set the model to be used for abilities.
      *
      * @param  string  $model
-     * @return $this
+     * @return void
      */
     public static function useAbilityModel($model)
     {
         Models::setAbilitiesModel($model);
-
-        return $this;
     }
```

Comment:
What's wrong here is that they were static in the first place, not that they returned `$this`.